src/main/java/hudson/tasks/test/TestObject.java

```diff
@@ -217,6 +218,7 @@ public AbstractTestResultAction getTestResultAction() {
      * Get a list of all TestActions associated with this TestObject. 
      */
     @Override
+    @Exported(visibility = 999)
```

Comment:
I would just leave the default visibility (3?) if Jenkins shows it in the API in such case.